modules/installation-configuration-parameters.adoc

```diff
@@ -3181,6 +3181,108 @@ The name of one or more failures domains.
 |The boot type that the compute machines use. You must use the `Legacy` boot type in {product-title} {product-version}. For more information on boot types, see link:https://portal.nutanix.com/page/documents/kbs/details?targetId=kA07V000000H3K9SAK[Understanding UEFI, Secure Boot, and TPM in the Virtualized Environment].
 |`Legacy`, `SecureBoot` or `UEFI`. The default is `Legacy`.
 
+|compute:
+  platform:
+    nutanix:
+      dataDisks:
+        dataSourceImage:
+          name:
+|Optional. The name of the storage container resource in Prism Central.
+|String
+
+|compute:
+  platform:
+    nutanix:
+      dataDisks:
+        dataSourceImage:
+          referenceName:
+|Optional. The identifier of the storage resource configured in the failure domain. If you use this identifier, you must configure a matching failure domain storage resource. For more information about configuring failure domains, see _Configuring failure domains_ in the _Installing a cluster on Nutanix_ page.
```

Comment:
Should be:

Optional. The identifier of the data source image in the failure domain. If you use this identifier, you must configure a matching dataSourceImage (with the same referenceName) in each failure domain the compute nodes will spread out. For more information about configuring failure domains, see _Configuring failure domains_ in the _Installing a cluster on Nutanix_ page.